libraries/exoplayer_rtsp/src/main/java/androidx/media3/exoplayer/rtsp/RtspMediaTrack.java

```diff
@@ -129,8 +130,15 @@ public int hashCode() {
         checkArgument(!fmtpParameters.isEmpty());
         processH265FmtpAttribute(formatBuilder, fmtpParameters);
         break;
+      case MimeTypes.AUDIO_RAW:
+        int pcmEncoding =
+            getPCMEncodingFromRtpMediaType(mediaDescription.rtpMapAttribute.mediaEncoding);
```

Comment:
Maybe add a local variable of `mediaDescription.rtpMapAttribute.mediaEncoding`, and substitute L107 too.